src/index.js

```diff
@@ -1,54 +1,56 @@
 'use strict';
 
-const $tagsInput = document.getElementById('tags-input');
-const $tagsList = document.getElementById('tags-list');
-const tagsArr = ['1', 'super tag'];
-
-$tagsInput.addEventListener('keyup', ({key, target}) => {
-  if (key === 'Enter' && target.value.trim()) {
-    const $el = createTagElement(target.value);
-    const isElementExist = tagsArr.includes(target.value);
-
-    if (!isElementExist) {
-      $tagsList.appendChild($el);
-      tagsArr.push(target.value);
-      target.value = '';
-    }
-  }
-});
-
-$tagsList.addEventListener('click', event => {
-  const {target} = event;
-  const isRemoveBtn = target.classList.contains('remove-btn');
-  const $foo = target.parentElement.parentElement;
-  const $tag = target.parentElement;
-
-  if (isRemoveBtn) {
-    $foo.removeChild($tag);
-
-    // TODO: remove element from array
-    // tagsArr.splice();
-  }
-});
+const $tagsInput = document.getElementsByClassName('tags-input');
+let index = 0;
 
 const createTagElement = content => {
   const $li = document.createElement('li');
-
   $li.className = 'tags-list-item';
   $li.innerHTML = `
     <span class="tag-name">${content}</span>
     <span class="remove-btn">x</span>
   `;
-
   return $li;
 };
 
-const init = () => {
-  tagsArr.forEach(tagValue => {
-    const $el = createTagElement(tagValue);
+[...$tagsInput].forEach(function (input) {
```

Comment:
you can use arrow function inside "forEach"
and you can use a second argument `function (input, index)` as a counter!
👎